src/utils/stringHashCode.ts

```diff
@@ -4,22 +4,22 @@
  * can be found in the LICENSE.txt file in the project root.
  */
 
-export const StringSeedHashCode = Math.round(Math.random() * Math.pow(2, 32));
+export const stringSeedHashCode = Math.round(Math.random() * Math.pow(2, 32));
```

Comment:
I find it useful to name constants like this one in all caps, but the naming convention linting rule doesn't allow it. I saw you had a //eslint-ignore for another naming earlier as well, I feel as a rule that potentially often will be disabled it can do more harm than good

Reply:
I believe there's no common standard in JS/TS for this, like in other languages. By nature this is a variable, which has to be named like that. There are cases where this does not apply (like a class import assigned to such a variable), but unfortunately ESLint does not allow to differentiate between such cases. 

The cases were the naming convention linter rule is disabled are those where const names, translated from Java, are used and should stay as they are to avoid confusion. I'd prefer not to suppress the warning, but I'm not aware of a way to handle those cases differently. So for now we stay with the suppression (where we want to keep the original name) or change the name to camelCase.

Reply:
Use casing like expected in JS/TS (here camelCase). Only for very specific cases deviate (and suppress the warning), like those all uppercase constants from Java.